File: gpao-tex-core/production/src/main/java/com/gpro/consulting/production/model/ProductionHeureDetails.java
package com.gpro.consulting.production.model;

import jakarta.persistence.*;
import lombok.*;

import java.util.Calendar;

@Entity
@Table(name = "gp_productionheure_destails")
@Data
@EqualsAndHashCode(callSuper = true)
@Builder
@AllArgsConstructor
@NoArgsConstructor
public class ProductionHeureDetails extends BaseEntity {

    @ManyToOne
    @JoinColumn(name = "gp_productionheure_id")
    private ProductionHeure productionHeure;

    @Id
    @GeneratedValue(generator = "PRODUCTION_HEURE_DETAILS_ID_GENERATOR", strategy = GenerationType.SEQUENCE)
    @SequenceGenerator(name = "PRODUCTION_HEURE_DETAILS_SEQ", sequenceName = "PRODUCTION_HEURE_DETAILS_SEQ", allocationSize = 1)
    private Long id;
    @Column(name = "id_details")
    private Long detailsId;
    private Double qteFinie;
    private Double qteNonEmballe;
    private Double qteRetouche;
    private Double qteDeuxiemeChoix;
    private Long nbreColis;
    private Calendar date;
    private Long numPaquet;
    @Column(name = "id_paquet")
    private Long paquetId;
    @Column(name = "id_colis")
    private Long colisId;
    private Boolean complement;
    private Long quantite;
    private String operation;
    private String observations;
    @Column(name = "id_taille")
    private Long tailleId;
    private Long couleurId;
    private Long ordrePaquet;
    private String numMatelasPaquet;
    private String nature;
}

File: gpao-tex-ui/source/main/src/app/services/production/models/production-heure-details.ts
/* tslint:disable */
/* eslint-disable */
import { ProductionHeure } from '../models/production-heure';
export interface ProductionHeureDetails {
  archived?: boolean;
  colisId?: number;
  complement?: boolean;
  couleurId?: number;
  creationDate?: string;
  creatorIdentifier?: string;
  date?: string;
  deletionDate?: string;
  detailsId?: number;
  id?: number;
  latestUpdaterId?: string;
  modificationDate?: string;
  nature?: string;
  nbreColis?: number;
  numMatelasPaquet?: string;
  numPaquet?: number;
  observations?: string;
  operation?: string;
  ordrePaquet?: number;
  paquetId?: number;
  productionHeure?: ProductionHeure;
  qteDeuxiemeChoix?: number;
  qteFinie?: number;
  qteNonEmballe?: number;
  qteRetouche?: number;
  quantite?: number;
  tailleId?: number;
  version?: number;
}
